src/redux/actions/cellMeta/loadCellMeta.js

```diff
@@ -30,6 +30,7 @@ const loadCellMeta = (
 
   const body = {
     name: plotWorkName[metaName],
+    config: {},
```

Comment:
why adding it now?

Reply:
Hmm... previously the API was complaining about requests not having `config` in it's body. I just checked the API validations for `mitochondrialContent` and `doubletScores` and they don't require config. I'll test it again and see the errors. I'll remove this if it's not needed.

Reply:
sounds good

Reply:
Refactored into one plot in 39ef247